apps/accounts/serializers.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
+
+from apps.users.models import UserProfile
+
+
+class UserProfileSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = UserProfile
+        fields = (
+            'username', 'display_name', 'email',
+            'bio', 'deleted', 'display_collections',
+            'display_collections_fav', 'homepage',
+            'location', 'notes', 'notifycompat',
+            'notifyevents', 'occupation', 'picture_type',
```

Comment:
Can you include the URL for the picture?

I feel like the list of fields could be trimmed down too. We shouldn't need `failed_login_attempts`, `last_login_attempt`, `last_login_attempt_ip` and `t_shirt_requested` (is that still active?). `notifyevents` and `notifycompat` can probably go away too but I don't know what they're for. They should at least get named `notify_events` or break out into another object like `notify: {events: true, compat: false}`.